test/small-integration-test.sh

```diff
@@ -42,7 +42,7 @@ export CACHE_DIR
 export ENSURE_DISKFREE_SIZE 
 if [ -n "${ALL_TESTS}" ]; then
     FLAGS=(
-        "use_cache=${CACHE_DIR} -o ensure_diskfree=${ENSURE_DISKFREE_SIZE} -o fake_diskfree=${FAKE_FREE_DISK_SIZE}"
+        "use_cache=${CACHE_DIR} -o ensure_diskfree=${ENSURE_DISKFREE_SIZE} -o fake_diskfree=${FAKE_FREE_DISK_SIZE} -o use_xattr"
```

Comment:
Should we have a separate flag for xattr instead of including it in the cache tests?

Reply:
_Sorry for my late reply_
It's best to have separate tests for xattr, but I don't think they affect each other when tested with other options.
(If we add another test pattern(option pair), the test time will be longer, so I don't think it's necessary to do that.)